### PR TITLE
Themeing

### PR DESCRIPTION
**Pull request by alangdm**
Initial draft for themeing

I tried to summarize the most recent thread + the previous conclusion on why ::theme was dropped in the first place.

I have a couple of questions that I added as comments to this PR, if anyone can clarify those please do so

### CODE REVIEW

File: reports/2022.html [hunk 1]
```diff
@@ -141,9 +141,9 @@ <h3>Table of Contents</h3>
             </tr>
             <tr>
               <th><a href="#themeing">Themeing</a></th>
+              <td><a href="https://github.com/WICG/webcomponents/issues/864">WICG/webcomponents#864</a></td>
               <td></td>
```
Comment (alangdm) [inline reports/2022.html:145]: What's the priority level for this?

File: reports/2022.html [hunk 2]
```diff
@@ -141,9 +141,9 @@ <h3>Table of Contents</h3>
             </tr>
             <tr>
               <th><a href="#themeing">Themeing</a></th>
```
Comment (alangdm) [inline reports/2022.html:143]: A bit late to ask this, but should it be "Themeing" or "Theming"?

File: reports/2022.html (hunk 1 above)
Comment (thescientist13) [reply reports/2022.html:145]: Yeah, I was thinking that maybe we [drop that column](https://github.com/w3c/webcomponents-cg/issues/43#issuecomment-1221625315) for this year's report?  In favor of our new outline per #53

File: reports/2022.html (hunk 2 above)
Comment (thescientist13) [reply reports/2022.html:143]: Yeah, it probably should be [the latter](https://en.wikipedia.org/wiki/Theming).

**Review by thescientist13 — APPROVED** [review]

**Comment by alangdm** [comment]
@thescientist13 for the time being I corrected the theming spelling, thanks for the clarification

**Comment by thescientist13** [comment]
@alangdm 
Could you do a quick merge conflict resolution when you get a chance?

**Comment by alangdm** [comment]
@thescientist13 
Done ✅

**Pull request merged**